freqtrade/optimize/backtesting.py

```diff
@@ -45,7 +45,7 @@
 ELONG_IDX = 6  # Exit long
 SHORT_IDX = 7
 ESHORT_IDX = 8  # Exit short
-BUY_TAG_IDX = 9
+ENTER_TAG_IDX = 9
 SHORT_TAG_IDX = 10
```

Comment:
This one should be `EXIT_TAG_IDX` to match right?

Reply:
no, there is no exit tag, if any, it's `LONG_TAG` and `SHORT_TAG`. 
but as mentioned in the other comment - i'm not convinced on having the 2 seperated will give us benefits.